outport/process/transactionsfee/utils.go

```diff
@@ -0,0 +1,67 @@
+package transactionsfee
+
+import (
+	"bytes"
+	"encoding/hex"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/smartContractResult"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"math/big"
+)
+
+const (
+	GasRefundForRelayerMessage = "gas refund for relayer"
+)
+
+func checkArg(arg ArgTransactionsFeeProcessor) error {
```

Comment:
this can be moved in the same file where the ArgTransactionsFeeProcessor is defined

Reply:
done